trg_ros2_node:
  ros__parameters:
    ros2:
      isDebug: true
      frameId: "world"
      publishRate: 10.0
      debugRate: 1.0
      topic:
        input:
          egoPose: "/trg/input/default_pose"
          egoOdom: "/odom"
          obsCloud: "/points_e1r_front"
          goal: "/fake_goal"
        output:
          preMap: "/trg/output/default_preMap"
          goal: "/trg/output/default_goal"
          path: "/trg/output/default_path"
        debug:
          globalTRG: "/trg/debug/default_globalTRG"
          localTRG: "/trg/debug/default_localTRG"
          obsMap: "/trg/debug/default_obsMap"
          pathInfo: "/trg/debug/default_pathInfo"
